lib/resources/dstu2/diagnostic_report.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Diagnostic Report
```

Comment:
Diagnostic Report => DiagnosticReport.
This would show the terminology binding as "DiagnosticReport.category" rather than "Diagnostic Report.category"